doc/changes/unreleased.md

```diff
@@ -1,9 +1,28 @@
 # Unreleased
 
+## Summary
+
+This release of the Notebook Connector comes with breaking changes in the `SlctManager`:
+* The class has been renamed to `ScriptLanguageContainer`.
+* Some public methods or and attributes have been
+  * renamed
+    * `flavor_name` to `flavor`
+    * `flavor_dir` to `flavor_path`
+    * `upload()` to `deploy()`
+    * `slc_docker_images()` to `docker_images()`
+    * `clean_all_images()` to `clean_docker_images()`
+  * removed
+    * `check_slc_repo_complete()`
+    * `clone_slc_repo()`
+* The handling of Secure Configuration Storage and the SlcSession have been changed.
```

Comment:
SLCSession didn't exist in the previous releases.

Reply:
See next push, updating the changelog